`spaced-comment` rule warns line-endings of Windows.

This is related with #2884.

On the issue, the problem around CRLF seems to be not fixed completely.
I tried with `master` branch, this rule still warn at normal block comments.

## test.js

```js
/* eslint spaced-comment:2 */

/*
 * test
 */

/**
 * test
 */
```

## Command & Actual

```
> git log -1
commit e88f8d816072957fcf9c67188ee609f3d40a4957
...
> node .\bin\eslint.js test.js --no-eslintrc

test.js
  3:1  error  Expected space or tab after /* in comment  spaced-comment

✖ 1 problem (1 error, 0 warnings)
```

## Expected

No errors.

----

To fix, I guess `&& node.value.indexOf("\r") !== 0` is added into [here](https://github.com/eslint/eslint/blob/e88f8d816072957fcf9c67188ee609f3d40a4957/lib/rules/spaced-comment.js#L86) merely.

--- a/lib/rules/spaced-comment.js
+++ b/lib/rules/spaced-comment.js
@@ -47,7 +47,7 @@ module.exports = function(context) {
         markers = unescapedMarkers.map(escaper);
 
         // the markerMatcher includes any markers in the list, followed by space/tab
-        markerMatcher = new RegExp("((^(" + markers.join("))|(^(") + ")))[ \\t\\n]");
+        markerMatcher = new RegExp("((^(" + markers.join("))|(^(") + ")))[ \\t\\r\\n]");
     }
 
     /**
@@ -83,7 +83,7 @@ module.exports = function(context) {
             }
 
             // Space expected and not found
-            if (node.value.indexOf(" ") !== 0 && node.value.indexOf("\t") !== 0 && node.value.indexOf("\n") !== 0) {
+            if (node.value.indexOf(" ") !== 0 && node.value.indexOf("\t") !== 0 && node.value.indexOf("\n") !== 0 && node.value.indexOf("\r") !== 0) {
 
                 /*
                  * Do two tests; one for space starting the line,

--- a/tests/lib/rules/spaced-comment.js
+++ b/tests/lib/rules/spaced-comment.js
@@ -132,6 +132,14 @@ eslintTester.addRuleTest("lib/rules/spaced-comment", {
             options: ["always"]
         },
         {
+            code: "/*\r\n *Test\r\n */",
+            options: ["never"]
+        },
+        {
+            code: "/*     \r\n *Test\r\n */",
+            options: ["always"]
+        },
+        {
             code: "/*!\n *comment\n */",
             options: ["always", { markers: ["!"] }]
         },

